Example/ViewController.swift

```diff
@@ -166,6 +168,14 @@ class ViewController: UIViewController {
     
     @IBAction func simulateButtonPressed(_ sender: Any) {
         simulationButton.isSelected = !simulationButton.isSelected
+        // When the simulation mode is on and the `LocationProvider` of the `MapView` is set up as `PassiveLocationManager`, the `NavigationMapView` should stop receiving location updates from the `PassiveLocationManager` and simply follow the simulated route during turn-by-turn navigation.
+        if simulationButton.isSelected {
+            navigationMapView.mapView.locationManager.locationProvider.stopUpdatingHeading()
+            navigationMapView.mapView.locationManager.locationProvider.stopUpdatingLocation()
+        } else {
+            navigationMapView.mapView.locationManager.locationProvider.startUpdatingHeading()
+            navigationMapView.mapView.locationManager.locationProvider.startUpdatingLocation()
+        }
```

Comment:
Pressing the button by itself should have no effect on how the preview map behaves. We should pause location updates when hiding the map in `uninstall(_:)`, since map location updates would be useless at that point, and resume location updates when reconfiguring the map in `configure(_:)`.

Reply:
The problem after check is that the `func uninstall(_ navigationMapView: NavigationMapView) ` is not called during the runtime. after remove the `navigationMapView` when turn-by-turn navigation start, it's fixed.